### DISCUSSION TITLE
sync-conflict files and Obsidian

### DISCUSSION THREAD

**Discussion by TonyWhitley** [MobiusSync/MobiusSync#169] [General]
When I edit an Obsidian note it creates many sync-conflict files separated by seconds.  In SyncThing I can mop these up fairly quickly but can't find the equivalent in MobiusSync.

Bonus marks to anyone who can tell me how to avoid creating the bloody things in the first place!

Edit: it seems they're bouncing back to my iPad from my PC.  I've set the filter to `**/*sync-conflict*` which may suppress them.

**Comment by SimonPickup** [comment]
You may wish to ask here: https://forum.obsidian.md/t/sync-mac-pc-and-ios-using-syncthing-mobius-sync/72022
